airbyte-webapp/src/components/connection/ConnectionForm/ConnectionConfigurationFormPreview.tsx

```diff
@@ -0,0 +1,104 @@
+import { useFormikContext } from "formik";
+import { FormattedMessage } from "react-intl";
+
+import { FlexContainer } from "components/ui/Flex";
+import { Text } from "components/ui/Text";
+
+import {
+  ConnectionScheduleData,
+  ConnectionScheduleType,
+  NamespaceDefinitionType,
+  NonBreakingChangesPreference,
+} from "core/request/AirbyteClient";
+import { FeatureItem, useFeature } from "hooks/services/Feature";
+
+import styles from "./ConnectionConfigurationFormPreview.module.scss";
+import { FormikConnectionFormValues } from "./formConfig";
+import { namespaceDefinitionOptions } from "./types";
+
+export const ConnectionConfigurationFormPreview: React.FC = () => {
+  const { getFieldMeta } = useFormikContext<FormikConnectionFormValues>();
```

Comment:
Since you type the `useFormikContext` I think you could pull this right out of the `values` object instead of making multiple `getFieldMeta` calls.

Reply:
Gosh! You are absolutely right 👍 Thanks!